### PR TITLE
Add specs for Docker::Container::ListResponse

### PR DESCRIPTION
**Pull request by owentams**
Basic tests to be sure Docker::Container::ListResponse properly parses the JSON sample from the documentation. Yet untested due to bugs in the current version of the origin branch

### CODE REVIEW

File: spec/docker/container/list_response.cr [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require "../../spec_helper"
+
+subject = Docker::Container::ListResponse.from_json ListResponseSample
+
+describe Docker::Container::ListResponse do
+  it "parses the sample data" do
+    subject.id.should eq "8dfafdbc3a40"
+    subject.names.should eq ["/boring_feynman"]
+    subject.image.should eq "ubuntu:latest"
+    subject.imageid.should eq "d74508fb6632491cea586a1fd7d748dfc5274cd6fdfedee309ecdcbc2bf5cb82"
```
Comment (dscottboggs) [inline spec/docker/container/list_response.cr]: imageid should be image_id

File: spec/docker/container/list_response.cr [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    subject.command.should eq "echo i"
+    subject.created.should eq 1367854155
+    subject.state.should eq "Exited"
+    subject.status.should eq "Exit 0"
+    subject.labels["com.example.vendor"].should eq "Acme"
```
Comment (dscottboggs) [inline spec/docker/container/list_response.cr]: need to add a test that labels isn't nil, then do the tests on a non-nil version

File: spec/docker/container/list_response.cr [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    subject.labels["com.example.license"].should eq "GPL"
+    subject.labels["com.example.version"].should eq "1.0"
+    subject.sizerw.should eq 12288
```
Comment (dscottboggs) [inline spec/docker/container/list_response.cr]: again, missing the underscores.

**Review by dscottboggs — CHANGES_REQUESTED** [review]
A few mistakes but otherwise good, thanks owen!

File: spec/docker/container/list_response.cr [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    subject.sizerootfs.should eq 0
+  end
+  describe Docker::Container::ListResponse::PortConfig do
+    it "parses the sample data" do
+      subject.ports[0].ip.should be_nil
+      subject.ports[1].private_port.should eq 2222
```
Comment (dscottboggs) [inline spec/docker/container/list_response.cr]: nope
